tracker_doctor.go

```
@@ -810,6 +834,97 @@ func isDirWritable(dir string) bool {
+	}
+	return hardStatus
+}
+
+// probeGitForDoctor performs the same two probes as pipeline.checkGit but
+// without reaching into the pipeline-internal helper. Local copy keeps the
+// doctor file's dependency surface clean.
+func probeGitForDoctor(workDir string) (installed bool, isRepo bool) {
+	if _, lerr := exec.LookPath("git"); lerr != nil {
+		return false, false
+	}
+	cmd := exec.Command("git", "-C", workDir, "rev-parse", "--git-dir")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)